tests/test_api.py

```diff
@@ -695,6 +740,38 @@ def test_is_delegated_role(self) -> None:
             self.assertFalse(role.is_delegated_path("a/non-matching path"))
             self.assertTrue(role.is_delegated_path("a/path"))
 
+    def test_zero_padding_in_succinct_roles(self) -> None:
+        bit_lengths = [2, 6, 8, 10, 16, 32]
+        expected_paddings = [1, 2, 2, 3, 4, 8]
+
+        for bit_length, expected in zip(bit_lengths, expected_paddings):
+            succinct_delegation = SuccinctRoles([], 1, bit_length, "foo")
+            m = f"Error for {bit_length} expected padding {expected}"
+            self.assertEqual(succinct_delegation.suffix_len, expected, m)
+
+    def test_is_delegated_role_in_succinct_roles(self) -> None:
+        succinct_roles = SuccinctRoles([], 1, 5, "bin")
+        # delegated role name suffixes are in hex format.
+        false_role_name_examples = ["foo", "bin-", "bin-s", "bin-20", "bin-100"]
+        for role_name in false_role_name_examples:
+            msg = f"Error for {role_name}"
+            self.assertFalse(succinct_roles.is_delegated_role(role_name), msg)
+
+        true_name_examples = ["bin-00", "bin-0f", "bin-1f"]
+        for role_name in true_name_examples:
+            msg = f"Error for {role_name}"
+            self.assertTrue(succinct_roles.is_delegated_role(role_name), msg)
+
+    def test_get_roles_in_succinct_roles(self) -> None:
+        succinct_roles = SuccinctRoles([], 1, 3, "bin")
+        bin_amount = 0
+        for i, role_name in enumerate(succinct_roles.get_roles()):
+            self.assertEqual(role_name, f"bin-{i}")
```

Comment:
This assert would become more interesting for a bit_length > 3, which would implicitly also test the padding and you wouldn't need `test_zero_padding_in_succinct_roles`.